src/main/java/pages/LoginPage.java

```diff
@@ -33,4 +39,96 @@ public String getPageTitle() {
         return driver.findElement(pageTitle).getText();
     }
 
-}
\ No newline at end of file
+    public boolean isTextPageTitleDisplayed() {
+        if (driver.findElement(pageTitle).isDisplayed()) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public String getRegisteredCustomerText() {
+        return driver.findElement(registeredCustomerText).getText();
+    }
+
+    public boolean isTextRegisteredCustomerDisplayed() {
+        if (driver.findElement(registeredCustomerText).isDisplayed()) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public String getSingInInstructionText() {
+        return driver.findElement(singInInstructionText).getText();
+    }
+
+    public boolean isTextSingInInstructionDisplayed() {
+        if (driver.findElement(singInInstructionText).isDisplayed()) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public String getEmailLabel() {
+        return driver.findElement(emailLabel).getText();
+    }
+
+    public boolean isTextEmailLabelDisplayed() {
+        if (driver.findElement(emailLabel).isDisplayed()) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public String getPasswordLabel() {
+        return driver.findElement(passwordLabel).getText();
+    }
+
+    public boolean isTextPasswordLabelDisplayed() {
+        if (driver.findElement(passwordLabel).isDisplayed()) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public String getSignInButtonText() {
+        return driver.findElement(signInButton).getText();
+    }
+
+    public boolean isSignInButtonEnabled() {
+        if (driver.findElement(signInButton).isEnabled()) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public String getForgotYourPassword() {
+        return driver.findElement(forgotYourPassword).getText();
+    }
+
+    public boolean isTextForgotYourPasswordDisplayed() {
+        if (driver.findElement(forgotYourPassword).isDisplayed()) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public String verifySignInButtonColor() {
+        WebElement SignInButton = driver.findElement(signInButton);
```

Comment:
In java cammelcase is used for defining variable name, starting with lowercase letter.
It should be signInButton.